test/pipe.js

```diff
@@ -76,8 +107,6 @@ if (process.platform !== 'win32') {
     });
   } else {
     console.error('Warning: Cannot verify piped exec');
+    shell.exit(123);
```

Comment:
Let's leave it, just to be safe. But I agree, `grep` is almost certainly available by default, and I would be extremely surprised if anyone intentionally deleted it. I've actually been meaning to refactor this to check for the existence of `grep` (windows or otherwise). Not necessary for this PR though.